kodilla-good-patterns/src/main/java/flightSearcher/FindFlight.java

```diff
@@ -48,6 +48,8 @@ public FindFlight(Flight flight) {
                     .distinct()
                     .forEach(finalList::add);
         }
-        return finalList;
+        return finalList.stream()
```

Comment:
Dalej nie rozumiem. 
W srode o 18.57 napisalem, jakie sygnatury maja miec metody. Po co to ja pisalem? Masz na hangoutsach.

Reply:
To znaczy, że zadanie nie jest wykonane bo metody mają złe sygnatury czy napisany kod nie robi tego, czego wymaga zadanie?